integration_tests/sdk/aqueduct_tests/param_test.py

```diff
@@ -512,3 +512,42 @@ def noop(input):
         expected_status=ExecutionStatus.FAILED,
         parameters={"number": "This is a string"},
     )
+
+
+def test_param_management(client):
+    # Create some implicit parameters that are consumed by downstream operators.
+    @op
+    def foo(param1, param2):
+        return 123
+
+    foo_output = foo(1000, "string val")
+
+    @op
+    def bar(input):
+        return input
+
+    bar_output = bar(foo_output)
```

Comment:
Since parameters 1 and 2 are already consumed by `foo`, we don't really need `bar` here for the tests to work right?

Reply:
yeah it's unnecessary. I just added it to make sure downstream operators would be removed too. I'll add that.